tests/test_split_docstrings.py

```diff
@@ -0,0 +1,41 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/normal")
+def doc_without_linefeed():
+    """
+    Normal doc without
+    line feed.
+    """
+    return {"id": "foo"}
+
+
+@app.get("/split")
+def doc_with_linefeed():
+    """
+    Documentation that is broken appart
+    \f
+    by a line feed. This part should not appear in the description.
+    """
+    return {"id": "foo"}
```

Comment:
I think the build will pass if you remove this return statement, and the one from the other endpoint above.

(FastAPI has a 100% coverage requirement for the *test code* as well.)